lstchain/reco/r0_to_dl1.py

```diff
@@ -600,20 +599,15 @@ def r0_to_dl1(
                         if not tag_pix_thr(image):
                             good_ring = False
                         else:
-                            # read geometry from event.inst. But not needed for every event. FIXME?
-                            geom = subarray.tel[telescope_id].\
-                                camera.geometry
-
                             muonintensityparam, dist_mask, \
                             ring_size, size_outside_ring, muonringparam, \
                             good_ring, radial_distribution, \
                             mean_pixel_charge_around_ring,\
                             muonpars = \
                                 analyze_muon_event(subarray,
-                                                   event.index.event_id,
-                                                   image, geom, focal_length,
-                                                   mirror_area, False, '')
-                            #                      mirror_area, True, './')
+                                                   tel_id, event.index.event_id,
+                                                   image, None, False, '')
```

Comment:
Can you add keyword arguments, at least for the constants here? It's hard to know what is set to `None`, `False` and `''`